suites/reef/rados/tier-3_rados_test-4-node-ecpools.yaml

```diff
@@ -1,6 +1,116 @@
 # Suite is to be used to verify the EC 2+2 config on 4 nodes
 
 tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574887
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                # using 6.1 rc candidate
+                custom_image: "registry-proxy.engineering.redhat.com/rh-osbs/rhceph:6-177"
+                custom_repo: "http://download-node-02.eng.bos.redhat.com/rhel-9/rel-eng/RHCEPH-6/RHCEPH-6.1-RHEL-9-20230602.0/compose/Tools/x86_64/os/"
```

Comment:
I currently do not know the reason behind hard-coding the build, however, this should still point to Reef rc candidate

Reply:
These were the paths added for upgrade. Let me know if we need to change this.

Quincy : 6.x GA -> 6.x Nightly
Reef : 6.x GA -> 7.x Nightly

Reply:
Updated to 7.0 RC build here.